CMakeLists.txt

```diff
@@ -1099,11 +1115,17 @@ ENDIF(PCRE2_BUILD_TESTS)
 # Installation
 
 SET(CMAKE_INSTALL_ALWAYS 1)
+set(TARGETS_EXPORT_NAME pcre2-targets)
 
 INSTALL(TARGETS ${targets}
-        RUNTIME DESTINATION bin
+        EXPORT ${TARGETS_EXPORT_NAME}
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
         LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
         ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+INSTALL(EXPORT ${TARGETS_EXPORT_NAME}
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/pcre2
+        NAMESPACE pcre2::
+        FILE ${TARGETS_EXPORT_NAME}.cmake)
```

Comment:
Redundant. This is the default file name for this export name.

Reply:
Removed.